apps/stats-web/env/.env.production

```
@@ -1,16 +1,9 @@
-NEXT_PUBLIC_NODE_ENV=$NODE_ENV
-NEXT_PUBLIC_BASE_API_MAINNET_URL=https://console-api.akash.network
-NEXT_PUBLIC_BASE_API_SANDBOX_URL=https://console-api-sandbox.akash.network
-NEXT_PUBLIC_BASE_API_TESTNET_URL=https://console-api-testnet.akash.network
-NEXT_PUBLIC_API_BASE_URL=$NEXT_PUBLIC_BASE_API_MAINNET_URL
+VITE_NODE_ENV=$NODE_ENV
+VITE_BASE_API_MAINNET_URL=https://console-api.akash.network
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_BASE_API_MAINNET_URL key should go before the VITE_NODE_ENV key

(UnorderedKey)